external/dhcpd/dhcpd.c

```diff
@@ -1637,7 +1639,7 @@ int dhcpd_run(void *arg)
 
 exit_with_error:
 	g_dhcpd_running = 0;
-
+    g_dhcp_sta_joined = 0;
```

Comment:
1. use a tab instead of spaces as indentation
2. Isn't it ```NULL```?

Reply:
Thanks, I'll fix it